src/components/courses/AdminPage.tsx

```
@@ -246,29 +291,79 @@ const AdminPage = () => {
-                        >
-                          <FaTrash />
-                        </button>
+                      <div className="flex justify-between w-full">
+                        <span>{exam.title}</span>
+                        <div className="space-x-2 flex items-center">
+                          <button
+                            onClick={() => toggleExamExpansion(exam.id)}
+                            className="text-gray-500 hover:text-gray-700 dark:text-gray-300 dark:hover:text-gray-100"
+                          >
+                            {expandedExam === exam.id ? (
+                              <FaChevronUp />
```

Biome

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)